content/docs/extensions/deployment/sagemaker.md

```diff
@@ -4,7 +4,114 @@ Implements MlemEnv, MlemDeployment and DeployState to work with AWS SageMaker
 
 ## Description
 
-**TODO**
+MLEM SageMaker allow you to deploy MLEM models to AWS SageMaker. You can learn
+more about SageMaker
+[here](https://docs.aws.amazon.com/sagemaker/latest/dg/whatis.html).
+
+### Preparing infrastructre
```

Comment:
typo., could you please run Grammarly or something on top of the text? (I see other typos, English syntax mistakes, etc)

Reply:
I use Code Spell Checker on VS Code and it helps fix typos, which is 80% of the problem.

I've just recently started trying Grammarly BTW and I'm not yet sure whether I can recommend it. But it can certainly help improve syntax marginally at least for some people.

Reply:
Also, I'd call this section
```suggestion
## Prerequisites
```
so it's more general (applicable to all pages)